src/commands/math/advancedSymbols.js

```diff
@@ -303,6 +303,9 @@ LatexCmds.alef = LatexCmds.alefsym = LatexCmds.aleph = LatexCmds.alephsym =
 LatexCmds.xist = //LOL
 LatexCmds.xists = LatexCmds.exist = LatexCmds.exists =
   bind(VanillaSymbol,'\\exists ','&exist;');
+  
+LatexCmds.nexists = LatexCmds.nexist =
+      bind(VanillaSymbol, '\\nexist ', '&#8708;');
```

Comment:
I think it's `\nexists`? http://mathb.in/54238

The HTML looks perfect though http://output.jsbin.com/kebili